styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,578 @@
+      .gh,
+      .gp {
+        color: @sapphire;
+      }
+      // Strings
+      .s,
+      .sa,
+      .sb,
+      .sc,
+      .dl,
+      .s2,
+      .sh,
+      .sd,
+      .s1,
+      .cpf {
+        color: @green;
+      }
+      .nc {
+        color: @yellow;
+      }
+      // Methods, Functions
+      .nf,
+      .fm {
+        color: @blue;
+      }
+      .nd,
+      .ow {
+        color: @mauve;
+      }
+      .nv,
+      .ss,
+      .vc,
+      .vg,
+      .vi,
+      .vm {
+        color: @text;
+      }
+      .si,
+      .sr {
+        color: @red;
+      }
+      pre {
+        color: @text !important;
+      }
+      border-color: @surface0 !important;
+    }
+    /* tables */
+    // help tab/page table
+    .mtable > tbody > tr:nth-child(even) {
+      background-color: @mantle !important;
+    }
+    // markdown tables
+    .ntable th {
+      background-color: @surface1 !important;
+      border-color: @surface2 !important;
+      color: @text;
+    }
+    .ntable tr:nth-child(even) {
+      background-color: @surface0 !important;
+    }
+  }
+}
+
+@-moz-document regexp('https?://rentry\\.(co|org)/.+/raw'),
+               url-prefix('https://rentry.co/static/'),
+               domain('export3.rentry.co') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+    body {
+      color: @text !important;
+      background: @base !important;
+    }
```

Comment:
Can you not just put this block in the first section and have it apply to this domains still? Or does it not work / break something?

Reply:
i don't think i understand your request correctly.
but i did this because there are some pages on the website that are just error pages/plain text ([/raw](https://rentry.co/how/raw), [/static/example](https://rentry.co/static/example)) so they dont have theming therefore no `.dark-mode` class therefore always defaulting to the light theme (`:root:not(.dark-mode)`). so this section makes it default to the device theme instead (`prefers-color-scheme`)